mypy_json_report/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Memrise
```

Comment:
I wasn't really sure what to do with the copyright years here. Should they be 2024? 2022-2024? As-is?

Reply:
It should be the year the project was established, so keep it at 2022.